sdk/cosmos/azure-cosmos/src/test/java/com/azure/cosmos/CosmosAsyncClientTest.java

```diff
@@ -46,7 +46,7 @@ public final void setTestName(Method method) {
         if (this.clientBuilder.getConnectionPolicy() != null && this.clientBuilder.configs() != null) {
             String connectionMode = this.clientBuilder.getConnectionPolicy().getConnectionMode() == ConnectionMode.DIRECT
                     ? "Direct " + this.clientBuilder.configs().getProtocol()
-                    : "Gateway";
+                    : this.clientBuilder.getEndpoint().contains(":8903") ? "ComputeGW" : "Gateway";
```

Comment:
Can we have this port number in a constant?

Reply:
Resolved in next iteration.